views/course_view/views.py

```diff
@@ -40,6 +39,8 @@ def get(self, request, course_code, semester_name):
         instructors = User.objects.filter(id__in=sections.values_list('instructor', flat=True), role='Instructor')
```

Comment:
This is OK. this view needs to be refactored to not use DB access but this is okay for now since it's outside of scope

Reply:
Yep haha, I didn't focused on course view.

Reply:
yea, i created a trello story for it